### PR TITLE
Pick and play random music

### PR DESCRIPTION
**Pull request by Marty99-dev**

### CODE REVIEW

**Review by marluxiaboss — APPROVED** [review]
It would be better if this was linked to the main menu to be able to test this functionality directly manually. Also there are hard coded strings that should reformatted as constants.

File: app/src/androidTest/java/ch/epfl/sdp/blindwar/game/GameTutorialTest.kt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+class GameTutorialTest {
+    // All possible musics during tutorial
+    private val expectedMusics = arrayOf(
+        MusicMetaData("Highway to Hell", "ACDC"),
+        MusicMetaData("Harder Better Faster Stronger", "Daft Punk"),
+        MusicMetaData("One More Time", "Daft Punk"),
+        MusicMetaData("Feel Good Inc", "Gorillaz"),
+        MusicMetaData("Poker Face", "Lady Gaga"),
+        MusicMetaData("Californication", "Red Hot Chili Peppers"),
+        MusicMetaData("Mistral gagnant", "Renaud"),
```
Comment (CHOCOLVT) [inline app/src/androidTest/java/ch/epfl/sdp/blindwar/game/GameTutorialTest.kt:22]: small incoherence with other music title format (All Words Begins With a Caps)

File: app/src/androidTest/java/ch/epfl/sdp/blindwar/game/GameTutorialTest.kt [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    )
+
+    @Test
+    fun testNextRound() {
+        //val gameTutorial = GameTutorial(InstrumentationRegistry.getInstrumentation().targetContext.assets)
+        val gameTutorial = GameTutorial(ApplicationProvider.getApplicationContext<Context>().assets)
+
+        // Iterate 10 times since we have 10 different musics in tutorial
+        val toPlay = expectedMusics.copyOf().toMutableSet()
+        for(i in 0..9){
```
Comment (CHOCOLVT) [inline app/src/androidTest/java/ch/epfl/sdp/blindwar/game/GameTutorialTest.kt:35]: magic number - but not important as Test

**Review by CHOCOLVT — APPROVED** [review]
generally good code

**Pull request merged**